src/main/java/it/gov/pagopa/payment/service/payment/qrcode/QRCodeAuthPaymentServiceImpl.java

```diff
@@ -37,7 +37,7 @@ public QRCodeAuthPaymentServiceImpl(TransactionInProgressRepository transactionI
   public AuthPaymentDTO authPayment(String userId, String trxCode) {
     TransactionInProgress trx = qrCodeAuthorizationExpiredService.findByTrxCodeAndAuthorizationNotExpired(trxCode.toLowerCase());
     if(trx == null){
-      throw new ClientExceptionWithBody(HttpStatus.NOT_FOUND,
+      throw new TransactionNotFoundOrExpiredException(
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor